cluster/kube-util.sh

```diff
@@ -94,6 +94,83 @@ function test-teardown {
 	echo "TODO: test-teardown" 1>&2
 }
 
+function find-kubectl-binary {
```

Comment:
It's not clear to my why we need this here? How have the scripts been finding kubectl up to now?